apps/docs/content/components/LogoSuite/index.mdx

```diff
@@ -18,7 +18,9 @@ export default ComponentLayout
 
 ## Usage
 
-Use the logo suite component to showcase a list of logos, such as for clients, sponsors, or related enterprises. Use the logo suite to build credibility, trust, and provide social proof by displaying consistent logos.
+Use the logo suite component to showcase a list of logos, such as for clients, sponsors, or related enterprises.
```

Comment:
These looked a bit too close in the docs, so I've separated onto two lines and removed repetition.